tests/integration/conftest.py

```diff
@@ -39,13 +10,19 @@ def splinter_webdriver():
 
 
 @pytest.fixture(scope='session')
-def title():
-    return random_string()
+def title(request):
```

Comment:
the random string solves the problem when these tests are being run in parallel on multiple different machines